src/workers/postBannedRep.ts

```diff
@@ -16,7 +16,8 @@ const worker: Worker = {
   subscription: 'post-banned-rep',
   handler: async (message, con, logger): Promise<void> => {
```

Comment:
We actually don't even check if the post is deleted or not here, but guess it's ok for now.

Reply:
We can maybe check in CDC directly and just ignore all deleted posts? That one would be easy to implement even in this PR?